dotnet-desktop-guide/framework/wpf/includes/xbap-unsupported.md

```diff
@@ -0,0 +1,9 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 03/11/2024
+ms.topic: include
+---
+
+>[!WARNING]
+> XBAPs require legacy browsers to operate, such as Internet Explorer and Firefox. These older browser versions are usually unsupported on Windows 10 and Windows 11. Modern browsers no longer support the technology required for XBAP apps due to security risks. Plugins that enable XBAPs are no longer supported.
```

Comment:
@TrevorFellman-MSFT Are you still around? I moved the previous note about XBAP to an include file and spread it throughout the articles related to XBAP. Can you check the wording here for me?

Reply:
@tdykstra This is the only change.